### PR TITLE
exploration graphql

### PR DESCRIPTION
**Pull request by Aayushitmdc**
This PR request contains content of the GraphQL page of the Exploration button on DPH.

### CODE REVIEW

File: docs/interfaces/data_product_hub/exploration.md [hunk 1]
```diff
@@ -567,5 +567,77 @@ For example, In the above image are the YAML Implementation files for the Lens m
 
 ### **GraphQL Tab**
 
-The GraphQL tab provides an interface for advanced users and developers to write and execute GraphQL queries. GraphQL enables precise data retrieval, allowing full control over the structure of the returned data. This is useful for performing complex or customized data extractions that extend beyond the standard capabilities of the Studio interface.
+The GraphQL tab provides an interface for advanced users and developers to programmatically interact with the data model. It displays the data returned by a GraphQL query alongside the query itself. This feature is useful for performing complex or customized data extractions that go beyond the standard capabilities of the Studio interface.
+
+The interface includes syntax highlighting,  autocompletion and automatic documentation and error highlighting. Users can utilize GraphiQL to interactively build both page and static queries.
```
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration.md:572]: There is an extra space before autocompletion

Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration.md:572]: Do not mention GraphiQL anywhere in the documentation

File: docs/interfaces/data_product_hub/exploration/graphql.png
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration/graphql.png:1]: The text shouldn't be written in this manner, change it, this type of font with background shadow is not allowed, ask @shraddaa how she does that

File: docs/interfaces/data_product_hub/exploration.md [hunk 2]
```diff
@@ -567,5 +567,77 @@ For example, In the above image are the YAML Implementation files for the Lens m
...
+
+Within the curly braces, press "Ctrl + Space" (or "Shift + Space" as an alternate keyboard shortcut) to open the autocomplete window. This will prompt a query for either logical tables or metrics in the data model. 
+
+<center>
+<img src="/interfaces/data_product_hub/exploration/graphql_autocomplete.png" alt="DPH" style="width:20rem; border: 1px solid black;" />
+</center>
+
+Construct a query like below:
+
+```
```
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration.md:606]: This particular code block doesn't have a defined language, which in this case is graphql

File: docs/interfaces/data_product_hub/exploration.md [hunk 3]
```diff
@@ -567,5 +567,77 @@ For example, In the above image are the YAML Implementation files for the Lens m
...
+      source
+    }
+  }
+}
+```
+
+**Components:**
+
+- Operation Type (query): Specifies the type of operation.
+- Operation Name (LensQuery): Optional name for the query, useful for debugging.
```
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration.md:619]: Is that an optional name?

File: docs/interfaces/data_product_hub/exploration.md [hunk 4]
```diff
@@ -567,5 +567,77 @@ For example, In the above image are the YAML Implementation files for the Lens m
...
+query LensQuery {
+  table {
+    sales {
+      source
+    }
+  }
+}
+```
+
+**Components:**
```
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration.md:616]: I think you have written these components in generic graphql terminology, write it in the context of DataOS

File: docs/interfaces/data_product_hub/exploration.md [hunk 5]
```diff
@@ -567,5 +567,77 @@ For example, In the above image are the YAML Implementation files for the Lens m
...
+- Operation Name (LensQuery): Optional name for the query, useful for debugging.
+- Root Field (table): The entry point defined in the Query root type.
+- Arguments (limit: 10): Parameters to customize the data retrieval.
+- Nested Fields (sales, total_revenue, source): Specifies the exact data to fetch.
+
+
+Press the **Execute** button (which resembles a "Play" icon) or use "Ctrl + Enter" to run the GraphQL query. The data will be returned in the Result window.
+
+
+As user query the model, the queries get saved and can be accessed by clicking on the history icon. The query will be saved with the name given in the query. This can also be accessed in the Saved query pane on top of the panels. (For e.g., here FirstQuery, SecondQuery is the qury name.)
```
Comment (pyjs-tmdc) [inline docs/interfaces/data_product_hub/exploration.md:628]: spelling of query is incorrect.

**Review by pyjs-tmdc — CHANGES_REQUESTED** [review]
@Aayushitmdc I have put comments in the changes that you have requested, please address them and make sure that the language, style guide, and grammar are according to what we had discussed.

File: docs/interfaces/data_product_hub/exploration.md (hunk 3 above)
Comment (Aayushitmdc) [reply docs/interfaces/data_product_hub/exploration.md:619]: Yes it is.

**Pull request merged**